batch/snippets/src/test/java/com/example/batch/CreateResourcesIT.java

```diff
@@ -59,7 +59,11 @@ public class CreateResourcesIT {
           + UUID.randomUUID().toString().substring(0, 7);
   private static final String CUSTOM_EVENT_NAME = "test-job"
           + UUID.randomUUID().toString().substring(0, 7);
-  private static final String JOB_ALLOCATION_POLICY_LABEL = "test-job"
+  private static final String BATCH_LABEL_JOB = "test-job-label"
+      + UUID.randomUUID().toString().substring(0, 7);
+  private static final String CUSTOM_NETWORK_NAME = "test-job-network"
+      + UUID.randomUUID().toString().substring(0, 7);
```

Comment:
Remove this.

Reply:
This code is in the master branch that was merged earlier. Two samples are tested using these variables. Why do I need to remove this code?

Reply:
I think this is a misunderstanding, please review my PR again. I don't have this piece of code as a new added. On the "Files Changed" page, I only see the code related to this PR.

Reply:
I think it was a dirty merge in the last commit. The latest one has  pulled in the changes.